polling_stations/templates/postcode_view.html

```diff
@@ -10,26 +10,6 @@
         {% stylesheet 'map' %}
     {% endif %}
 
-    <style>
-        .council_location_icon {
-            background-color: black;
-            border-radius: 50%;
-            height: 2px;
-            width: 2px;
-        }
-
-        address {
-            margin-top: 1rem;
-            margin-bottom: 1rem;
-            font-size: inherit;
-            line-height: 1.6;
-            text-rendering: optimizeLegibility;
-        }
-
-        .callout.success {
-            margin-top: 2rem;
-        }
-    </style>
```

Comment:
I'm not 100% sure this is redundant, but I'm also not 100% sure how to check. When I delete it nothing changes, but perhaps it's for mobile or something? Any ideas?

Reply:
@VirginiaDooley Reckons these are no longer in use.  Apart from in a template that is dropped  in 0941b62b9f3b3.